src/geocoder.js

```diff
@@ -37,7 +37,7 @@ export default class Geocoder {
     }
 
     this.timeBetweenRequests =
-      options.clientId && options.privateKey || options.apiKey ? 100 : 200;
+      options.clientId && options.privateKey || options.apiKey ? 20 : 200;
```

Comment:
Thinking about this: When you always need a key or id, it is always 20. So we can remove this check and make it a constant.